bench/commands/__init__.py

```diff
@@ -70,4 +70,10 @@ def bench_command(bench_path='.'):
 bench_command.add_command(remote_urls)
 
 from bench.commands.install import install
-bench_command.add_command(install)
\ No newline at end of file
+bench_command.add_command(install)
+
+from bench.commands.exclude_update import exclude_update
```

Comment:
Import exclude_update and include_update like 
`from bench.commands.exclude_update import exclude_update, include_update`